plugins/direct_events_plugin/db/services.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$services = array(
+      'directeventhandler' => array(                                                // the name of the web service
+          'functions' => array ('local_direct_event_handler'), // web service functions of this service
+          'requiredcapability' => '',                // if set, the web service user need this capability to access 
+                                                                              // any function of this service. For example: 'some/capability:specified'                 
+          'restrictedusers' => 0,                                             // if enabled, the Moodle administrator must link some user to this service
+                                                                              // into the administration
+          'enabled' => 1,                                                       // if enabled, the service can be reachable on a default installation
+          'shortname' =>  'd_handler',       // optional – but needed if restrictedusers is set so as to allow logins.
+          'downloadfiles' => 0,    // allow file downloads.
+          'uploadfiles'  => 0,      // allow file uploads.
+          'ajax' => true
+       )
+  );
+
+  $functions = array(
+    'local_direct_event_handler' => array(         //web service function name
```

Comment:
Since we're currently implementing this to just process individual content loaded events, what about calling this `local_fe_events_direct_content_loaded_handler`?